provider/infoblox/infoblox.go

```diff
@@ -253,50 +274,58 @@ func (p *ProviderConfig) Records(ctx context.Context) (endpoints []*endpoint.End
 
 		var resC []ibclient.RecordCNAME
 		objC := ibclient.NewEmptyRecordCNAME()
-		objC.View = p.view
-		objC.Zone = zone.Fqdn
-		err = p.client.GetObject(objC, "", nil, &resC)
+		err = p.client.GetObject(objC, "", queryParams, &resC)
 		if err != nil && !isNotFoundError(err) {
-			return nil, fmt.Errorf("could not fetch CNAME records from zone '%s': %s", zone.Fqdn, err)
+			return nil, fmt.Errorf("could not fetch CNAME-records from zone %q: %s", zone.Fqdn, err.Error())
 		}
 		for _, res := range resC {
-			logrus.Debugf("Record='%s' CNAME:'%s'", res.Name, res.Canonical)
+			logrus.Debugf("Record=%q CNAME:%q", res.Name, res.Canonical)
 			endpoints = append(endpoints, endpoint.NewEndpoint(res.Name, endpoint.RecordTypeCNAME, res.Canonical))
 		}
 
 		if p.createPTR {
-			// infoblox doesn't accept reverse zone's fqdn, and instead expects .in-addr.arpa zone
-			// so convert our zone fqdn (if it is a correct cidr block) into in-addr.arpa address and pass that into infoblox
+			// Infoblox NIOS doesn't accept reverse zone's fqdn, and instead expects .in-addr.arpa zone
+			// so convert our zone fqdn (if it is a correct cidr block) into in-addr.arpa address and pass that to Infoblox NIOS
 			// example: 10.196.38.0/24 becomes 38.196.10.in-addr.arpa
 			arpaZone, err := transform.ReverseDomainName(zone.Fqdn)
 			if err == nil {
-				var resP []ibclient.RecordPTR
+				var resPtrStatic, resPtrDynamic []ibclient.RecordPTR
```

Comment:
Same questions as [in the previous PR](https://github.com/kubernetes-sigs/external-dns/pull/2841#discussion_r987665022).